extension/js/common/browser.ts

```diff
@@ -1239,6 +1239,14 @@ export class AttUI {
     return atts;
   }
 
+  clearAtt = (id: string) => {
+    delete this.attachedFiles[id];
+  }
```

Comment:
This method is a carbon copy of `cancelAtt`

Reply:
`cancelAtt` is a private method, I didn't want to change the access key of your methods. But I can change and use `cancelAtt` if you would prefer. Or I could call `cancelAtt` from with in `clearAtt`.

Reply:
Since I am just clearing all attachments I will remove this method.

Reply:
Please go ahead and change cancelAtt to public, that is better than creating effectively the same method that is public.

You should not need `cancelAtts` next to it, the already existing method should do what you need.